pkg/spec/createconfig.go

```diff
@@ -417,12 +412,14 @@ func (c *CreateConfig) GetContainerCreateOptions(runtime *libpod.Runtime) ([]lib
 
 		options = append(options, libpod.WithIPCNSFrom(connectedCtr))
 	}
-	if IsPod(string(c.IpcMode)) {
-		options = append(options, libpod.WithIPCNSFromPod(pod))
-	}
```

Comment:
So these three removed appends for options aren't needed?

Reply:
Yes we are handling the pod configuration in podman not in libpod.